pkg/sql/testing.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2019 The SQLFlow Authors. All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sql
```

Comment:
seems `getEnv` is not used?

Reply:
`getEnv` is still in use by code in this branch, so, while I am removing `log.go`, where `getEnv` is defined, I had to move `getEnv` to this new file `testing.go`. But we will likely be able to remove this `testing.go` after the merge of https://github.com/sql-machine-learning/sqlflow/pull/1494

Reply:
I see, thanks~